spec/models/route_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe Route do
+  describe 'route validations' do
+    subject { route.valid? }
+
+    let(:route) do
+      build(
+        :route,
+        name: name,
+        length_of_time: length_of_time
+      )
+    end
+    let(:name) { 'some name' }
+    let(:length_of_time) { 100 }
+
+    it { is_expected.to be_truthy }
+
+    subject { build(:route) }
```

Comment:
dont put two subjects in a block.
subject should always be first.

how about we change this whole test file to something like:
```
require 'rails_helper'

describe Route do
  describe 'route validations' do
    subject { build(:route) }

    it { is_expected.to be_valid }
    it { is_expected.to validate_presence_of(:name) }
    it { is_expected.to validate_presence_of(:length_of_time) }
    it 'validates numericality' do
      expect(subject).to validate_numericality_of(:length_of_time).
        is_greater_than_or_equal_to(1).
        is_less_than_or_equal_to(500)
    end
  end
end
```

Reply:
this wont be necessary anymore?
```
let(:name) { 'some name' }
let(:length_of_time) { 100 }

it { is_expected.to be_truthy }

Reply:
we are going to change `it { is_expected.to be_truthy }` into `it { is_expected.to be_valid }`
so that we can use a different `subject`
and since we arent using the variables defined here:
```
let(:name) { 'some name' }
let(:length_of_time) { 100 }
```
we can just do away with it